docs/src/main/sphinx/release/release-377.md

```diff
@@ -0,0 +1,29 @@
+# Release 377 (13 Apr 2022)
+
+## General
+
+* Add support for ANSI compliant `trim` function. ({issue}`11236`)
+* Fix bug when in certain rare cases fault-tolerant execution of queries
+  returned incorrect results. ({issue}`11870`)
+
+## Hive connector
+
+* Improve performance of queries that use Glue metadata. ({issue}`11869`)
+* Fix `sync_partition_metadata` procedure when partition names differ from
```

Comment:
What's fixed? Failure? Incorrect results?

Reply:
From reading the PR this could have caused problems with partition names including escaped characters..